### ISSUE TITLE
[FIX] [a11y] Missing form label

### ISSUE THREAD

**Issue by sonya0504** [infermedica/component-library#512]
## Describe the bug
Missing form label for text input

## To Reproduce
1. Open the modal in storybook
2. Open canvas in new tab 
3. Use Wave plugin to evaluate a11y

## Expected behavior
There should be no missing labels and no issues pointed by the Wave plugin.

## Screenshots
![image](https://github.com/user-attachments/assets/71485cdd-83b7-4072-a2e5-224fbd7588b3)
